.github/workflows/nightly-test-integrations-1.17.x.yml

```diff
@@ -319,6 +408,7 @@ jobs:
     - generate-envoy-job-matrices
     - envoy-integration-test
     - upgrade-integration-test
```

Comment:
should this be removed?

Reply:
Since we haven't migrated all tests to the new framework, we still need to keep them (though I have deleted a duplicate test case in another PR you reviewed last week)

Reply:
FYI: https://github.com/hashicorp/consul/pull/19653